mscore/CMakeLists.txt

```diff
@@ -612,8 +612,8 @@ if (MINGW)
       ${MINGW_ROOT}/lib/libvorbis.dll
       ${MINGW_ROOT}/lib/libvorbisfile.dll
       ${MINGW_ROOT}/opt/bin/ssleay32.dll
-      ${QT_INSTALL_PREFIX}/bin/libEGL.dll
-      ${QT_INSTALL_PREFIX}/bin/libGLESv2.dll
+      ${QT_INSTALL_BINS}/bin/libEGL.dll
+      ${QT_INSTALL_BINS}/bin/libGLESv2.dll
```

Comment:
If `${QT_INSTALL_LIBS} includes `/lib` I'd expect `${QT_INSTALL_BINS}` to include `/bin`. Wrong?

Reply:
Yes, looks like a mistake, thank you.